server/@types/Handover.ts

```diff
@@ -1,9 +1,12 @@
+import { CriminogenicNeedsData } from './Assessment'
+
 export type HandoverContextData = {
   handoverSessionId: string
   principal: HandoverPrincipal
   subject: HandoverSubject
   assessmentContext: HandoverAssessmentContext
   sentencePlanContext: HandoverSentencePlanContext
+  criminogenicNeedsData?: CriminogenicNeedsData
```

Comment:
Is the `criminogenicNeedsData` field not always suppled? I think there's times where the contents might be empty/undefined, but I think the field might always be there

Reply:
im not sure if its always going to be supplied, so just handling nulls in case